poseidon/Cargo.toml

```diff
@@ -38,9 +42,11 @@ std = [
     "ark-sponge/std",
     "poseidon-parameters/std",
     "poseidon-permutation/std",
+    "poseidon-paramgen/std",
```

Comment:
meh, that's sad that we have to include it also for library in `std` :| but that's better than introducing `build-binary-std` feature I guess

Reply:
I'm not so sure you need it here - instead opt in for the default feature (which is the) when adding the paramgen `dependency` - it's always used in std context anyway